src/App.tsx

```diff
@@ -1,11 +1,5 @@
 import { Store } from './types/index';
-import {
-  RouterProvider,
-  BrowserRouter,
-  Route,
-  createRoutesFromElements,
-  createBrowserRouter,
-} from 'react-router-dom';
+import { RouterProvider, BrowserRouter, Route, createRoutesFromElements, createBrowserRouter } from 'react-router-dom';
```

Comment:
@camilovegag, please check if we can make an exception for imports to have singleAttributePerLine